### DISCUSSION TITLE
Allow setting uneditable Markdown in issue forms

### DISCUSSION THREAD

**Discussion by Josh-Cena** [community/community#7045] [Projects and Issues]
Related to #4599, but that one's going in the wrong direction by assuming that `Markdown` elements would be showing up in the submitted issue, which may cause this valid request go away unnoticed.

I wish there's a way to set uneditable Markdown which does appear in the submitted issue. This could be as simple as allowing `contentEditable: false` in the `attributes` field of `textArea` and `input`. The most obvious use-case is allowing me to set a title for the issue template.

**Comment by lfrancke** [comment]
I'd like this as well to include checklists for certain types of issues automatically.
This way whoever works on a ticket has a checklist to work on, ideally this'd even be hidden in the displayed form

**Comment by mnapoli** [comment]
Here is another use case we have: we want to communicate to users that:

- we prioritize feature requests based on community feedback
- users are encouraged to add 👍  reactions to the issue

So we want to add a fixed paragraph at the top of every issue, e.g.:

> Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.

Right now with issue form it doesn't look like it's possible unfortunately, is that correct?

**Comment by SirSpidey** [comment]
Pulling together related requests for renderable Markdown:https://github.com/orgs/community/discussions/63402#discussioncomment-10341167
